### PR TITLE
Readme nits

### PR DESCRIPTION
**Pull request by richardc**
Testing code review, and fixing things in the docs.
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/parisiale/libpuppetdb/pull/1%23issuecomment-48720953%22%2C%20%22https%3A//github.com/parisiale/libpuppetdb/pull/1%23issuecomment-48721559%22%2C%20%22https%3A//github.com/parisiale/libpuppetdb/pull/1%23issuecomment-48721591%22%2C%20%22https%3A//github.com/parisiale/libpuppetdb/pull/1%23discussion_r14816765%22%2C%20%22https%3A//github.com/parisiale/libpuppetdb/pull/1%23discussion_r14816773%22%2C%20%22https%3A//github.com/parisiale/libpuppetdb/pull/1%23discussion_r14816783%22%2C%20%22https%3A//github.com/parisiale/libpuppetdb/pull/1%23discussion_r14816860%22%2C%20%22https%3A//github.com/parisiale/libpuppetdb/pull/1%23discussion_r14816880%22%2C%20%22https%3A//github.com/parisiale/libpuppetdb/pull/1%23discussion_r14816927%22%2C%20%22https%3A//github.com/parisiale/libpuppetdb/pull/1%23discussion_r14817024%22%2C%20%22https%3A//github.com/parisiale/libpuppetdb/pull/1%23discussion_r14817040%22%2C%20%22https%3A//github.com/parisiale/libpuppetdb/pull/1%23discussion_r14817051%22%2C%20%22https%3A//github.com/parisiale/libpuppetdb/pull/1%23discussion_r14817052%22%2C%20%22https%3A//github.com/parisiale/libpuppetdb/pull/1%23discussion_r14817186%22%2C%20%22https%3A//github.com/parisiale/libpuppetdb/pull/1%23issuecomment-48722309%22%2C%20%22https%3A//github.com/parisiale/libpuppetdb/pull/1%23issuecomment-48722387%22%2C%20%22https%3A//github.com/parisiale/libpuppetdb/pull/1%23discussion_r14821921%22%2C%20%22https%3A//github.com/parisiale/libpuppetdb/pull/1%23discussion_r14821955%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/parisiale/libpuppetdb/pull/1%23issuecomment-48720953%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22This%20is%20a%20global%20testing%20comment%21%22%2C%20%22created_at%22%3A%20%222014-07-11T11%3A41%3A02Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7150922%3F%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/parisiale%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222014-07-11T11%3A50%3A12Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7150922%3F%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/parisiale%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%3Agun%3A%22%2C%20%22created_at%22%3A%20%222014-07-11T11%3A50%3A29Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7150922%3F%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/parisiale%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222014-07-11T12%3A00%3A57Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7150922%3F%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/parisiale%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ahammer%3A%20time%21%22%2C%20%22created_at%22%3A%20%222014-07-11T12%3A02%3A06Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/55650%3F%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/richardc%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20a7c9f7d01bf23bd88b4055ff8018229bb2c55d57%20README.md%2049%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/parisiale/libpuppetdb/pull/1%23discussion_r14821921%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22What%20about%20Invincible%3F%22%2C%20%22created_at%22%3A%20%222014-07-11T14%3A04%3A25Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7150922%3F%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/parisiale%22%7D%7D%2C%20%7B%22body%22%3A%20%22Yes%2C%20let%27s%20get%20rid%20of%20all%20mythological%20beasts%22%2C%20%22created_at%22%3A%20%222014-07-11T14%3A05%3A03Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/55650%3F%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/richardc%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20README.md%3AL30-61%22%7D%2C%20%22Pull%20a7c9f7d01bf23bd88b4055ff8018229bb2c55d57%20README.md%209%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/parisiale/libpuppetdb/pull/1%23discussion_r14816773%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Ok%2C%20I%20agree%20here.%22%2C%20%22created_at%22%3A%20%222014-07-11T11%3A42%3A21Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7150922%3F%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/parisiale%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222014-07-11T11%3A49%3A30Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7150922%3F%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/parisiale%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20README.md%3AL1-16%22%7D%2C%20%22Pull%20a7c9f7d01bf23bd88b4055ff8018229bb2c55d57%20README.md%2016%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/parisiale/libpuppetdb/pull/1%23discussion_r14816783%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Again%21%5Cr%5CnBut%20tell%20me%20if%20when%20you%20read%20this.%22%2C%20%22created_at%22%3A%20%222014-07-11T11%3A42%3A47Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7150922%3F%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/parisiale%22%7D%7D%2C%20%7B%22body%22%3A%20%22I%20can%20read%20reel%20gud%22%2C%20%22created_at%22%3A%20%222014-07-11T11%3A46%3A13Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/55650%3F%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/richardc%22%7D%7D%2C%20%7B%22body%22%3A%20%22Try%20some%20more%21%22%2C%20%22created_at%22%3A%20%222014-07-11T11%3A47%3A08Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7150922%3F%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/parisiale%22%7D%7D%2C%20%7B%22body%22%3A%20%22Ok%2C%20ok...%22%2C%20%22created_at%22%3A%20%222014-07-11T11%3A53%3A44Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7150922%3F%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/parisiale%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222014-07-11T11%3A54%3A09Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7150922%3F%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/parisiale%22%7D%7D%2C%20%7B%22body%22%3A%20%22Actually%20talking%20to%20the%20voices%20in%20my%20head%2C%20we%20should%20call%20it%20%28cycle%20%27PuppetDB%27%20%27puppet%20DB%27%20%27squirrel%27%29%22%2C%20%22created_at%22%3A%20%222014-07-11T12%3A00%3A19Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/55650%3F%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/richardc%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20README.md%3AL1-16%22%7D%2C%20%22Pull%20a7c9f7d01bf23bd88b4055ff8018229bb2c55d57%20README.md%204%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/parisiale/libpuppetdb/pull/1%23discussion_r14816765%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Contractions%20are%20good%21%22%2C%20%22created_at%22%3A%20%222014-07-11T11%3A41%3A55Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7150922%3F%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/parisiale%22%7D%7D%2C%20%7B%22body%22%3A%20%22Maybe%20having%20the%20whole%20word%20is%20better...%22%2C%20%22created_at%22%3A%20%222014-07-11T11%3A53%3A08Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7150922%3F%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/parisiale%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-complete%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222014-07-11T11%3A54%3A09Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7150922%3F%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/parisiale%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20README.md%3AL1-16%22%7D%7D%2C%20%22approved%22%3A%20%7B%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/parisiale/libpuppetdb/pull/1#issuecomment-48720953'>General Comment</a></b>
- <a href='https://github.com/parisiale'></a> This is a global testing comment!
- <a href='https://github.com/parisiale'></a> :shipit:
- <a href='https://github.com/parisiale'></a> :shipit::gun:
- <a href='https://github.com/parisiale'></a> :+1:
- <a href='https://github.com/richardc'></a> :hammer: time!
- [ ] <a href='#crh-comment-Pull a7c9f7d01bf23bd88b4055ff8018229bb2c55d57 README.md 16'></a> &nbsp;<b><a href='https://github.com/parisiale/libpuppetdb/pull/1#discussion_r14816783'>File: README.md:L1-16</a></b>
- <a href='https://github.com/parisiale'></a> Again!
  But tell me if when you read this.
- <a href='https://github.com/richardc'></a> I can read reel gud
- <a href='https://github.com/parisiale'></a> Try some more!
- <a href='https://github.com/parisiale'></a> Ok, ok...
- <a href='https://github.com/parisiale'></a> :+1:<a href='#crh-update-none'></a>
- <a href='https://github.com/richardc'></a> Actually talking to the voices in my head, we should call it (cycle 'PuppetDB' 'puppet DB' 'squirrel')
- [ ] <a href='#crh-comment-Pull a7c9f7d01bf23bd88b4055ff8018229bb2c55d57 README.md 49'></a> &nbsp;<b><a href='https://github.com/parisiale/libpuppetdb/pull/1#discussion_r14821921'>File: README.md:L30-61</a></b>
- <a href='https://github.com/parisiale'></a> What about Invincible?
- <a href='https://github.com/richardc'></a> Yes, let's get rid of all mythological beasts
- [x] <a href='#crh-comment-Pull a7c9f7d01bf23bd88b4055ff8018229bb2c55d57 README.md 4'></a> &nbsp;<b><a href='https://github.com/parisiale/libpuppetdb/pull/1#discussion_r14816765'>File: README.md:L1-16</a></b>
- <a href='https://github.com/parisiale'></a> Contractions are good!
- <a href='https://github.com/parisiale'></a> Maybe having the whole word is better...
- [x] <a href='#crh-comment-Pull a7c9f7d01bf23bd88b4055ff8018229bb2c55d57 README.md 9'></a> &nbsp;<b><a href='https://github.com/parisiale/libpuppetdb/pull/1#discussion_r14816773'>File: README.md:L1-16</a></b>
- <a href='https://github.com/parisiale'></a> Ok, I agree here.

<a href='https://www.codereviewhub.com/parisiale/libpuppetdb/pull/1?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/parisiale/libpuppetdb/pull/1?approve=1'></a>&nbsp;<a href='https://github.com/parisiale/libpuppetdb/pull/1'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by parisiale** [comment]
This is a global testing comment!

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # libpuppetdb
 
-## Intro
+## Introduction
```
Comment (parisiale) [inline README.md:3]: Contractions are good!

File: README.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 
-libpuppetdb is a simple library for the execution of Puppet DB queries.
-The library uses libcurl and relies on the Puppet DB REST API.
+libpuppetdb is a simple library for the execution of PuppetDB queries.
+The library uses libcurl and relies on the PuppetDB REST API.
```
Comment (parisiale) [inline README.md:6]: Ok, I agree here.

File: README.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 Both both SSL (HTTP) and non-SSL (HTTPS) queries are supported.
 
 ## Using libpuppetdb
 
 To use libpuppetdb in your code, you first create a PuppetdbConnector object.
-You do that by specifying the Puppet DB host and, for secure connections, the
+You do that by specifying the PuppetDB host and, for secure connections, the
```
Comment (parisiale) [inline README.md:12]: Again!
But tell me if when you read this.

Comment (richardc) [inline README.md:12]: I can read reel gud

Comment (parisiale) [inline README.md:12]: Try some more!

File: README.md (hunk 2 above)
Comment (parisiale) [inline README.md:6]: :+1:

**Comment by parisiale** [comment]
:shipit:

**Comment by parisiale** [comment]
:shipit::gun:

File: README.md (hunk 1 above)
Comment (parisiale) [inline README.md:3]: Maybe having the whole word is better...

File: README.md (hunk 3 above)
Comment (parisiale) [inline README.md:12]: Ok, ok...

Comment (parisiale) [inline README.md:12]: :+1:<a href='#crh-update-none'></a>

File: README.md (hunk 1 above)
Comment (parisiale) [inline README.md:3]: :+1:<a href='#crh-update-complete'></a>

File: README.md (hunk 3 above)
Comment (richardc) [inline README.md:12]: Actually talking to the voices in my head, we should call it (cycle 'PuppetDB' 'puppet DB' 'squirrel')

**Comment by parisiale** [comment]
:+1:

**Comment by richardc** [comment]
:hammer: time!

File: README.md [hunk 4]
```diff
@@ -30,25 +30,31 @@ connection or query.
...
 
-## Build Requirements
+## Build requirements
 
 libpuppetdb is written in C++11, so you must call your compiler accordingly.
 
 The libcurl dev library is required. It must be SSL-enabled for secure
 connections.
 
+## Unicorns
```
Comment (parisiale) [inline README.md]: What about Invincible?

Comment (richardc) [inline README.md]: Yes, let's get rid of all mythological beasts

**Pull request closed without merging**